### ISSUE TITLE
Deploy Contract does not work

### ISSUE THREAD

**Issue by okayu8** [neo-project/neo-gui#272]
I tried to deploy contract.
I fulfilled all the items that required input, I set avm file and clicked Deploy button.

Invoke is not displayed on the screen after that.
Clicking the Test button will also pop up the following sentences.
`Excecution terminated in fault state.`

And Results shows the following.
```
VM State: FAULT, BREAK
Gas Consumed: 500
Evaluation Stack: []
```

Compiler's compatible setting failed regardless of whether it was true or false.

I thought it was fixed at the following URL, but I got this NEO-GUI on 2019/02/08.

Do you know the cause?

**Comment by okayu8** [comment]
Then I cloned the version of neo-gui with the latest version and used it, I was able to press the button of invoke.
(4572ec0d5bce0f1b4a59fc2e5cc3313a3ef3bcfb)

But a new problem occurred.

When clicking the test button at the time of deployment, the following is displayed as the result.

```
VM State: HALT, BREAK
Gas Consumed: 500
Evaluation Stack: [{"type": "InteropInterface"}]
```

After that, when you click the invoke button, Transacton successful is displayed and items created in Transaction History are added.
However, even if Confirm of this new item has passed for about 1 hour, it remains as uncinfirm.
No matter how many times I tried, it did not change from unconfirm.

Do you understand this cause?

**Comment by jsolman** [comment]
Please ensure it is including enough gas for the size of the transaction.

**Comment by jsolman** [comment]
I have a PR out on neo-cli that has deploy contract support that includes enough gas.

**Comment by erikzhang** [comment]
Hi @jsolman , https://github.com/neo-project/neo-cli/pull/273 has conflicts that need be resolved.

**Comment by jsolman** [comment]
I’ll fix them soon.

**Comment by jsolman** [comment]
@erikzhang conflicts were resolved for neo-project/neo-cli#273

**Comment by okayu8** [comment]
@jsolman Thank you, I will try this.
